vendor/github.com/google/go-cmp/cmp/path.go

```diff
@@ -315,7 +315,7 @@ func (tf Transform) Option() Option { return tf.trans }
 // pops the address from the stack. Thus, when traversing into a pointer from
 // reflect.Ptr, reflect.Slice element, or reflect.Map, we can detect cycles
 // by checking whether the pointer has already been visited. The cycle detection
-// uses a seperate stack for the x and y values.
+// uses a separate stack for the x and y values.
```

Comment:
Hmm any reason to touch the `vendor` file?

Reply:
Oh good catch, I didn't realize that was a vendor file.